packages/financial-templates-lib/src/clients/InsuredBridgeL2Client.ts

```diff
@@ -22,6 +22,11 @@ export interface Deposit {
   depositContract: string;
 }
 
+type BlockSearchConfig = {
+  fromBlock: number;
```

Comment:
Should we allow filter params so the caller can filter events?

Reply:
Sure done